### DISCUSSION TITLE
site blocked

### DISCUSSION THREAD

**Discussion by lverna** [microsoft/jacdac#1343] [Q&A]
I have the jacdac starter kit and microbit v2.

my students are blocked from downloading the blocks needed to code the jacdac accessories because github is blocked for them. 

I can't get my IT dept. to unblock them. do you have any suggestions or ways to work around this?

I am not blocked from github as a teacher.

thanks!!

Luke

**Comment by pelikhan** [comment]
> my students are blocked from downloading the blocks needed to code the jacdac accessories because github is blocked for them.

Could you detail the steps you use to download the jacdac blocs? MakeCode should automatically serve them to you through makecode.com to avoid the github.com blocked issues.

As for the simulator, unfortunately, we do not have another host than https://microsoft.github.io/jacdac-docs/ for now.

**Comment by lverna** [comment]
Hello,


I go to https://makecode.microbit.org/ and click on extensions.  then i search for jacdac and try to add the blocks and I get this message (as a student).  You can see in the lower left corner our filter blocks the students.  Is there another way to download the blocks?


thanks!!


Luke


[cid:054ef4ee-69b7-4cec-a674-b54e6fa1592f]


Luke Verna
he/him/his
Grades 6-8 Computer Technology Teacher
Upper Perkiomen Middle School
901 Montgomery Avenue
Pennsburg PA 18073 USA
267.313.4800 ext 6119
***@***.***


________________________________

**Reply by jamesadevine** [reply]
Can you screen shot the specific error and "lower left corner filter"?

**Comment by lverna** [comment]
is this what you are looking for?

[cid:63e1c8fc-0baf-4362-a545-6b740bfad7ce]


Luke Verna
he/him/his
Grades 6-8 Computer Technology Teacher
Upper Perkiomen Middle School
901 Montgomery Avenue
Pennsburg PA 18073 USA
267.313.4800 ext 6119
***@***.***


________________________________

**Reply by jamesadevine** [reply]
No I am after a screenshot of what you are seeing on the student account - it would also be useful to have a screenshot of console error output from makecode as this would indicate which network requests are being blocked.

**Comment by lverna** [comment]
I'm not sure what you are asking, sorry. I only see what i sent in the screenshot when signed in as a student and try to add jacdac blocks to the makecode site.  We use google accounts to sign in.


I'll keep asking my IT dept. to open the site


thanks!!


Luke Verna
he/him/his
Grades 6-8 Computer Technology Teacher
Upper Perkiomen Middle School
901 Montgomery Avenue
Pennsburg PA 18073 USA
267.313.4800 ext 6119
***@***.***


________________________________

**Reply by jamesadevine** [reply]
You so far haven't sent a screenshot (see above on github).

**Reply by lverna** [reply]
![jacdac error message](https://github.com/microsoft/jacdac/assets/59697187/2826d07d-f172-464e-904c-8f6823c1ebe0)

**Reply by lverna** [reply]
sorry, i was replying through email

**Reply by jamesadevine** [reply]
@pelikhan I guess because the simulator is served through github.io the add blocks button is similarly not available?

**Reply by FrankSchmidtDDS** [reply]
In my tests, off-line (which is the only way that I can simulate a public school fire-wall against Github) even when the blocks are installed in a saved .hex file with Jacdac modules the system fails. It appears that the module blocks still try to refer back to Github in order to activate. I get the 'rain cloud' image in simulator when not on-line even with installed modules. Also, if I click "button A" on the Microbit to "show number-Jacdac sensor value"  where that sensor block is installed but off-line, the return on the Microbit led array is "?". Unless I'm mistaken, from the point of view of a school teacher where there are fire-walls for student protection, this appears to be an unsolvable  problem. Any assistance would be much appreciated.
